tests/Command/PullRequest/PullRequestPatOnTheBackCommandTest.php

```diff
@@ -40,7 +44,72 @@ function (HelperSet $helperSet) {
         );
     }
 
-    private function getTemplateHelper()
+    public function testPatOnTheBackWithOption()
+    {
+        $tester = $this->getCommandTester(
+            new PullRequestPatOnTheBackCommand(),
+            null,
+            null,
+            function (HelperSet $helperSet) {
+                $helperSet->set($this->getTemplateHelper('thanks'));
+            }
+        );
+
+        $tester->execute(['pr_number' => 10, '--pat' => 'thanks'], ['interactive' => false]);
+
+        $display = $tester->getDisplay();
+
+        $this->assertCommandOutputMatches(
+            'Pat on the back pushed to https://github.com/gushphp/gush/pull/10',
+            $display
+        );
+    }
+
+    public function testChoosePatOnTheBack()
+    {
+        $command = new PullRequestPatOnTheBackCommand();
+        $tester = $this->getCommandTester(
+            $command,
+            null,
+            null,
+            function (HelperSet $helperSet) {
+                $helperSet->set($this->getTemplateHelper('beers'));
+            }
+        );
+
+        $this->setExpectedCommandInput(
+            $command,
+            [
+                'beers'
+            ]
+        );
+
+        $tester->execute(['pr_number' => 10]);
+
+        $display = $tester->getDisplay();
+
+        $this->assertCommandOutputMatches(
+            'Pat on the back pushed to https://github.com/gushphp/gush/pull/10',
+            $display
+        );
+    }
+
+    public function testPatOnTheBackWithWrongOptions()
+    {
+        $command = new PullRequestPatOnTheBackCommand();
+        $tester = $this->getCommandTester(
+            $command
```

Comment:
why this is 3 lines instead of just one, even L99 can be removed $command and you can do everything into one line